akka-persistence-typed/src/test/java/jdocs/akka/persistence/typed/BasicPersistentBehaviorsTest.java

```diff
@@ -35,12 +38,12 @@ public State emptyState() {
 
     @Override
     public CommandHandler<Command, Event, State> commandHandler() {
-      return (ctx, state, command) -> Effect().none();
+      return (ctx, state, command) -> someEffectFromCommand;
     }
 
     @Override
     public EventHandler<Event, State> eventHandler() {
-      return (state, event) -> state;
+      return (state, event) -> newStateAfterEvent;
     }
```

Comment:
I'm not sure about these changes. The purpose of the snippet is to have a copy-paste template to start out with, and this will not compile and will raise the question of what `someEffectFromCommand` is. Provided by Akka or something I have to implement, or is the documentation wrong since it doesn't compile.

What do you think about adding `TODO` comment instead?

Reply:
I'd be fine with TODOs, so in that case I think we need to return `null` in those cases for Java, and ??? for Scala because I wouldn't want it to silently do the wrong thing—i.e. it needs to be implemented. Wdyt?

Reply:
sounds good, perhaps `throw RuntimeException("an implementation is missing")` instead of `null`

Reply:
@patriknw How about: `throw new RuntimeException("TODO: return the next state")` etc?

Reply:
that's good

Reply:
@patriknw I just pushed an update which uses RuntimeExceptions as we discussed. Have a look at your convenience.